pkgs/applications/misc/schemes/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
```

Comment:
As a non-binding advice, put this list in ASCIIbetical order (after fetchFromGitLab).